src/definitions.jl

```diff
@@ -561,3 +576,58 @@ Pre-plan an optimized real-input unnormalized transform, similar to
 the same as for [`brfft`](@ref).
 """
 plan_brfft
+
+##############################################################################
+
+region(p::Plan) = p.region
+region(p::ScaledPlan) = region(p.p)
```

Comment:
Same here, this is not tested it seems (but tests are part of the `region` PR).

Reply:
Rebased that PR into this instead